tutorials/vr/developing_for_oculus_quest.rst

```diff
@@ -0,0 +1,126 @@
+.. _doc_developing_for_oculus_quest:
+
+Introduction
+------------
+
+This tutorial goes over how to get started developing for the
+Oculus Quest with an official plugin.
+
+First there are two things you need to do.
+
+First you need to go through the steps on the :ref:`doc_exporting_for_android`
+page. This leads you through installing the toolset that Godot
+needs to export to Android devices.
+
+Next you need the Quest Plugin. You can download it from the Asset
+Library or manually download it from `here <https://github.com/GodotVR/godot-oculus-mobile-asset>`__.
+
+Setting Up Godot
+----------------
+
+To get started Open up Godot and create a new project
+
+.. image:: img/quest_new_project.png
+
+Make sure to chose the GLES2 renderer. Due to optimisations in the
+Quest's GPU this backend is far better suited for the Quest.
+
+Copy the addons folder from the Oculus Mobile asset into your Godot
+project. Your project tree should looks something like this:
+
+.. image:: img/quest_project_tree.png
+
+Now we can start building our main scene. To keep things simple
```

Comment:
[Writing Guidelines](http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html)  suggest not using words like simple. What is simple to you may be black magic to me and vice versa